validation.rst

```diff
@@ -214,13 +214,13 @@ file:
 
     .. code-block:: yaml
 
-        # app/config/config.yml
+        # config/packages/validation.yaml
```

Comment:
the changes will probably have to happen in `config/packages/framework.yaml`

Reply:
indeed, thanks.